data/tribes/initialization/atlanteans/units.lua

```diff
@@ -1473,45 +1508,80 @@ descriptions:new_tribe {
             -- TRANSLATORS: Purpose helptext for an atlantean production site: Hunter's House
             purpose = pgettext("building", "Hunts animals to produce meat."),
             -- TRANSLATORS: Note helptext for an atlantean production site: Hunter's House
-            note = pgettext("atlanteans_building", "The hunter’s house needs animals to hunt within the work area.")
+            note = pgettext("atlanteans_building", "The hunter’s house needs animals to hunt within the work area."),
+            -- TRANSLATORS: Lore helptext for an atlantean production site: Hunter's House
+            lore = pgettext("atlanteans_building", "Sometimes we end up in a region with no fish, so we must hunt, "..
+                                                   "Game will reproduce by itself, if not hunted to much. But yes, fish is our main meal."),
+            -- TRANSLATORS: Lore author helptext for an atlantean production site: Hunter's House
+            lore_author = pgettext("atlanteans_building", "Hunter answering a fisher")
          }
       },
       {
          name = "atlanteans_well",
          helptexts = {
             -- TRANSLATORS: Purpose helptext for an atlantean production site: Well
-            purpose = pgettext("building", "Draws water out of the deep.")
+            purpose = pgettext("building", "Draws water out of the deep."),
+            -- TRANSLATORS: Note helptext for an atlantean production site: Well
+            note = pgettext("atlanteans_building", "I hope you checked this spot with a gelogist."),
+            -- TRANSLATORS: Lore helptext for an atlantean production site: Well
+            lore = pgettext("atlanteans_building", "Well, there is no wellbeing without a well."),
+            -- TRANSLATORS: Lore author helptext for an atlantean production site: Well
+            lore_author = pgettext("atlanteans_building", "Water carrier muttering to himself")
          }
       },
       {
          name = "atlanteans_gold_spinning_mill",
          helptexts = {
             -- TRANSLATORS: Purpose helptext for an atlantean production site: Gold Spinning Mill
-            purpose = pgettext("atlanteans_building", "Spins gold thread out of gold.")
+            purpose = pgettext("atlanteans_building", "Spins gold thread out of gold."),
+            -- TRANSLATORS: Note helptext for an atlantean production site: Gold Spinning Mill
+            note = pgettext("atlanteans_building", "Do not build until you have a steady gold supply."),
+            -- TRANSLATORS: Lore helptext for an atlantean production site: Gold Spinning Mill
+            lore = pgettext("atlanteans_building", "No, this is not Rumpelstiltskin spinning gold from straw, we need real gold"),
+            -- TRANSLATORS: Lore author helptext for an atlantean production site: Gold Spinning Mill
+            lore_author = pgettext("atlanteans_building", "Painted on a Gold Spinning mill without supplies.")
```

Comment:
```suggestion
            lore_author = pgettext("atlanteans_building", "Painted on a gold spinning mill without supplies")
```
Excuse me but did you actually *apply* the diff I provided and the suggestions I made, or did you just look over it and hand-picked _one_ changed character for every line in which I annotated _multiple_ spelling/grammar errors.

Reply:
No I did nit apply it as I changed some wordings on my way through the diff. sorry fo the extra effords